pandas/tests/test_expressions.py

```diff
@@ -21,13 +21,7 @@
 
 
 if not expr._USE_NUMEXPR:
-    try:
-        import numexpr  # noqa
-    except ImportError:
-        msg = "don't have"
-    else:
-        msg = "not using"
-    raise nose.SkipTest("{0} numexpr".format(msg))
+    numexpr = pytest.importorskip('numexpr')
```

Comment:
I meant that previously the test was also skipped if _USE_NUMPEXPR is false, now only when the import fails